Harvest.Core/Services/InvoiceService.cs

```diff
@@ -42,12 +42,13 @@ public async Task<bool> CreateInvoice(int id)
             }
 
             var now = DateTime.UtcNow;
-            if (project.Start > now)
+            if (project.Start.AddDays(30) > now) //Start doing invoices 30 days after the project starts
             {
                 //Project hasn't started yet. (Is Start UTC? if not, it should be)

```

Comment:
(I believe start is UTC but we only let them choose date so it doesn't really make a difference)